policykit/policyengine/models.py

```diff
@@ -512,10 +512,16 @@ def shouldCreate(self):
         return not self.pk # Runs only when object is new
 
     def save(self, *args, **kwargs):
+        kwargs_clean = kwargs.copy()
+        for key in 'treat_as_new', 'without_govern':
+            if kwargs.get(key):
+                kwargs_clean.pop(key)
+
         """
         Saves the action. If new, checks it against current passed policies. Note: Only meant for internal use.
         """
-        if self.shouldCreate():
```

Comment:
I'm guessing the point of this was so that subclasses could override it like this
https://github.com/amyxzhang/policykit/blob/da19825490164e770b83c9339a3775ae02bd28d9/policykit/policyengine/models.py#L751-L752
@amyxzhang or @rickygv99 would know more

Reply:
Yes, for deferring policy evaluation, the `shouldCreate` function should be overridden.